cmd/argocd/commands/utils/prompt.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"github.com/argoproj/argo-cd/v2/pkg/apiclient"
+	"github.com/argoproj/argo-cd/v2/util/cli"
+	"github.com/argoproj/argo-cd/v2/util/localconfig"
+)
+
+type Prompt struct {
+	enabled bool
+}
+
+func NewPrompt(clientOpts *apiclient.ClientOptions) (*Prompt, error) {
```

Comment:
I've decided to pass in `clientOpts *apiclient.ClientOptions` here to reduce duplicated set up code and to make clear to other contributors that the boolean value for the prompts-enabled setting is supposed to come from local config

If we instead passed through a `bool` value as the argument, it may not be clear to other developers where this value comes from

Reply:
Yes, thats make sense, but i would just apply config path instead whole structure. Because here a lot of redundant fields in clients options that not relevant for prompts